doc/source/tutorials/animations.rst

```diff
@@ -0,0 +1,150 @@
+.. _animations:
+
+.. index:: animations
+
+How to create an animation using Emperor
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In this tutorial we describe how to create a principal coordinates analysis
+(PCoA) plot, and display animated traces of the samples sorted by a metadata
+category.
+
+To do this, we need to have two metadata categories, a *gradient* category, and
+a *trajectory* category. The *gradient* category determines the order in which
```

Comment:
Have we always call it trajectory? IMOO it sounds confusing cause the trajectory is given by the gradient, right? Perhaps grouping is a better name but OK if this is how we always have call it.